projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-saved-cart.ts

```diff
@@ -590,50 +593,92 @@ export function updateSavedCartAndRestore(
 
         alerts.getSuccessAlert().should('contain', `Cart Edited Successfully`);
 
-        // restore saved cart
-        const restoreSavedCartAlias = interceptRestoreSavedCartEndpoint(
-          cart.code
-        );
-        const getAllSavedCartAlias = interceptGetAllSavedCartEndpoint();
+        if (addToActiveCart) {
+          cy.intercept(
+            'POST',
+            `${Cypress.env('OCC_PREFIX')}/${Cypress.env(
+              'BASE_SITE'
+            )}/${Cypress.env(
+              'OCC_PREFIX_USER_ENDPOINT'
+            )}/*/carts/*/entries?*lang=en&curr=USD`
+          ).as('add_to_cart');
```

Comment:
```suggestion
          const addToCartHandler = postToAddToCart()
```
Can it be extracted to a function?